x-pack/plugins/observability_solution/observability/public/routes/routes.tsx

```diff
@@ -155,4 +159,18 @@ export const routes = {
     params: {},
     exact: true,
   },
+  [OLD_SLO_DETAIL_PATH]: {
+    handler: () => {
+      return <SimpleRedirect to="/:sloId" redirectToApp="slo" />;
```

Comment:
Don't you need something before `/:sloId`? How does the `<SimpleRedirect />` component distinguish between the edit and detail route? The `to` prop is the same in both cases right?